pandas/core/arrays/datetimes.py

```diff
@@ -1576,6 +1643,24 @@ def sequence_to_dt64ns(data, dtype=None, copy=False,
     # if dtype has an embedded tz, capture it
     tz = validate_tz_from_dtype(dtype, tz)
 
+    if isinstance(data, (ABCSeries, ABCIndexClass)):
+        data = data._data
```

Comment:
unpacking Series is already done up on L1634.  Why dos it need to be done again?